README.md

```diff
@@ -1 +1,63 @@
 # Tiidelab-Cohort-4
+> A personal professional portfolio of Victor Ade-Samuel
```

Comment:
- [ ] Kindly update your project description current one says the app is a portfolio app.